Documentation/GettingStarted.md

```diff
@@ -8,16 +8,31 @@ There are 3 steps to getting started with the logging framework:
 2.  Configure the framework.
 3.  Convert your NSLog statements to use the Lumberjack macros
 
-### Add the lumberjack files to your project
+### Add LumberJack to your project
 
-The main files you need to add are:
+#### CocoaPods
 
--   DDLog (Basis of entire framework)
--   DDASLLogger (sends log statements to Apple System Logger, so they show up on Console.app)
--   DDTTYLogger (sends log statements to Xcode console - if available)
--   DDFileLogger (sends log statements to a file)
+```ruby
+platform :ios, '6.1'
```

Comment:
This was copied from README.md. If it should be 5.0 (I'm not sure) it needs changing in both.